code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -379,19 +393,32 @@
 	new /obj/item/reagent_containers/food/pill/fakedeath(src)
 	new /obj/item/reagent_containers/food/pill/fakedeath(src)
 
-/obj/item/storage/pill_bottle/patch_pack/ert
-	name = "ert patch pack"
-	desc = "A patch pack containing medical patches. Issued to Nanotrasen ERT medics."
+/obj/item/storage/pill_bottle/patch_pack/ert_red
```

Comment:
this should be left as 
```suggestion
/obj/item/storage/pill_bottle/patch_pack/ert
```
or /obj/item/storage/pill_bottle/patch_pack/ert/populate_contents() wont work.